content/docs/guide.md

```diff
@@ -356,11 +356,17 @@ since we try to use all the latest features.
 
 ## Check that Packit works
 
-With the configuration above, Packit is able only to build your changes and provide them as RPMs.
-Once you create a pull request, Packit builds the code from the
-pull request in Copr repository and reports back Copr build status to the pull request.
-
-An example of Packit pull request:
+With the configuration above, Packit will build your changes and provide them as RPMs.
+Once you create a pull request, Packit will:
+1. Check out the PR.
+2. Merge it with the main branch (this can be configured by the
+   [`merge_pr_in_ci`](https://packit.dev/docs/configuration/#merge_pr_in_ci)
+   configuration option).
+3. Submit a build with the code from the pull request in Copr [in its
```

Comment:
Is this true? My understanding is that it will submit a build with the code resulting from the merge, not the code in the PR (that's probably what you meant, but the wording is misleading).

Reply:
@pcahyna thank you! both of your suggestions definitely make the explanation more clear